apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/exception/SimpleChallengeGlobalExceptionTest.java

```diff
@@ -0,0 +1,92 @@
+package org.sagebionetworks.openchallenges.organization.service.exception;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+public class SimpleChallengeGlobalExceptionTest {
+
+  @Test
+  public void SimpleChallengeGlobalException_ShouldReturnMessage_WhenMessageKeyIsPassed() {
+    // Create an instance of SimpleChallengeGlobalException using the constructor with details
+    String details = "Something went wrong";
+    SimpleChallengeGlobalException exception = new SimpleChallengeGlobalException(details);
+
+    // Verify the exception details
+    assertThat(exception.getMessage()).isEqualTo(details);
+  }
+
+  @Test
+  public void SimpleChallengeGlobalException_ShouldReturnType_WhenTypeKeyIsPassed() {
+    // Define the exception details
+    String type = "ExceptionType";
+    String title = "Exception Title";
+    HttpStatus status = HttpStatus.BAD_REQUEST;
+    String detail = "Exception detail message";
+
+    // Create an instance of SimpleChallengeGlobalException using the all-args constructor
+    SimpleChallengeGlobalException exception =
+        new SimpleChallengeGlobalException(type, title, status, detail);
+
+    // Verify the exception details
+    assertThat(exception.getType()).isEqualTo(type);
+  }
+
+  @Test
+  public void SimpleChallengeGlobalException_ShouldReturnTitle_WhenTitleKeyIsPassed() {
+    // Define the exception details
+    String type = "ExceptionType";
+    String title = "Exception Title";
+    HttpStatus status = HttpStatus.BAD_REQUEST;
+    String detail = "Exception detail message";
+
+    // Create an instance of SimpleChallengeGlobalException using the all-args constructor
+    SimpleChallengeGlobalException exception =
+        new SimpleChallengeGlobalException(type, title, status, detail);
```

Comment:
This code seems to be the same as in the previous function. In that case, we should consolidate the two tests into a single one that include multiple `assertThat`. This will make the tests complete faster as there are less operations to do. Also, code duplication is bad for several reason and is one of the metrics that SonarCloud tracks.

Reply:
Condensed test with similar code as requested